tests/cursorid/cursorid-debug-003.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+MongoDB\Driver\CursorID debug output on 64-bit platform
+--SKIPIF--
+<?php if (4 == PHP_INT_SIZE) { die('skip Only for 64-bit platform'); } ?>
```

Comment:
For the sake of consistency with other tests, I'd suggest `8 !== PHP_INT_SIZE`.

As a side note, the PHP driver/library also aims to use strict equality operators (i.e. `===` and `!==`) whenever possible instead of the basic `==` and `!=` operators. The strict operators require types to match, while the basic operators allow for type juggling. This is discussed a bit more in [Comparison Operators](https://www.php.net/manual/en/language.operators.comparison.php). For some insight into why type juggling can be a bad thing, you can read [PHP Sadness: Comparison Operators](http://phpsadness.com/sad/52).

One exception is when we're dealing with comparing stdClass objects, since strict equality requires both operands to be the same object in memory. See [this paste](https://3v4l.org/2B1vV) for an example. In cases where we're using separate stdClass instances to represent BSON documents and we just want to compare them, we might defer to `==` to do a match since the objects are certainly not the same in memory.